toonz/sources/toonzlib/cleanupcolorstyles.cpp

```diff
@@ -90,9 +90,9 @@ void TCleanupStyle::setColorParamValue(int index, const TPixel32 &color) {
 QString TCleanupStyle::getParamNames(int index) const {
   switch (index) {
   case 0:
-    return "Brightness";
+    return QObject::tr("Brightness");
   case 1:
-    return "Contrast";
+    return QObject::tr("Contrast");
```

Comment:
Thank you for the modification, it's very helpful.
Can you please make "HRange" and "Line Width" translatable as well? They are in `TColorCleanupStyle::getParamNames()` in [lines 227-231](https://github.com/opentoonz/opentoonz/blob/792dc2b6b402d622292efe815bfaa2635c43eeb9/toonz/sources/toonzlib/cleanupcolorstyles.cpp#L227-L231) of `cleanupcolorstyles.cpp`.

Reply:
Thanks for your time to review and the kind words.
"HRange" and "Line Width" are added now.